pages/glossary/focused.md

```diff
@@ -7,4 +7,6 @@ input_aspects:
   - CSS styling
 ---
 
-An element is _focused_ when that element matches the [`:focus` pseudo-class](https://drafts.csswg.org/selectors-4/#focus-pseudo).
+An element is said to be _focused_ when the element matches the [`:focus` pseudo-class](https://drafts.csswg.org/selectors-4/#focus-pseudo) uninterruptedly for a period of 5 seconds after a user interacted with the page.
```

Comment:
I'd rather say "without a user interacting with the page" or something similar, since the user could keep interacting

🤔 

Actually, maybe we don't care about it.

Reply:
I like your suggestion. I think it captures the intention better.